mithril-client/examples/mithril-client-snapshot-example/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "mithril-client-snapshot-example"
+description = "Mithril client snapshot example"
+version = "0.1.0"
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+anyhow = "1.0.75"
+async-trait = "0.1.73"
+futures = "0.3.28"
+indicatif = "0.17.7"
+# To implement `mithril-client` crate in your own project, remove the line below and run `cargo add mithril-client`
+mithril-client = { path = "../../../mithril-client" }
```

Try this and remove the comment if it works:
```suggestion
mithril-client = { path = "../../../mithril-client", version="0.5.7"}
```